frontend/__tests__/components/utils/name-value-editor.spec.tsx

```diff
@@ -49,7 +49,7 @@ describe(NameValueEditor.displayName, () => {
         />
       );
 
-      expect(wrapper.html()).not.toContain('pairs-list__add-icon');
+      expect(wrapper.html()).not.toContain('annotations__add-icon');
```

Comment:
I prefer the existing name.  This isn't an annotation.  @spadgett?

Reply:
reverted to existing name

Reply:
> I prefer the existing name. This isn't an annotation. @spadgett?

+1, thanks